src/librustc_mir/const_eval.rs

```diff
@@ -697,8 +697,9 @@ pub fn const_eval_raw_provider<'tcx>(
                 // promoting runtime code is only allowed to error if it references broken constants
                 // any other kind of error will be reported to the user as a deny-by-default lint
                 _ => if let Some(p) = cid.promoted {
+                    use crate::interpret::InvalidProgramInfo::*;
```

Comment:
Is `ReferencedConstant` the only thing that's used? If yes, please just import that.